backend/app/services/knowledge/permission_request_service.py

```
@@ -0,0 +1,630 @@
+
+        Returns:
+            Updated permission request
+
+        Raises:
+            ValueError: If validation fails or permission denied
+        """
+        # Get the permission record
+        permission = db.query(Permission).filter(Permission.id == request_id).first()
+
+        if not permission:
+            raise ValueError("Permission request not found")
```

Ruff (0.14.14)

Avoid specifying long messages outside the exception class

(TRY003)